### DISCUSSION TITLE
Hentai Subtitles

### DISCUSSION THREAD

**Discussion by cmcglownjr** [stashapp/stash#3874] [Q&A] [answered]
How do I get the subtitles to display when watching hentai videos in Stash? The file types I usually mkv files and you would have to select the subtitle track in the media player if they aren't enabled by default. Some files have the subtitles embedded like my mp4 files. Where in the settings do I enable subtitle tracks. Thanks

**Comment by DogmaDragon** [comment]
Stash only supports external captions at the moment. Details https://docs.stashapp.cc/in-app-manual/captions/

**Reply by cmcglownjr** [reply]
Thank you for answering. Is there plans for embedded caption support in the future?

**Reply by DogmaDragon** [reply, marked as answer]
Haven't seen an issue for it, so you can create one. It's low priority one, so it's more likely to get implemented if someone who cares about it codes it itself and submits a pull request.

**Reply by stg-annon** [reply]
I do believe there was some kind of issue and that it was easier to pull them out of the files, I'd have to go digging to find out why

**Reply by feederbox826** [reply]
Issue was opened here https://github.com/stashapp/stash/issues/3875
